plugins/org.wso2.developerstudio.eclipse.distribution.project/src/org/wso2/developerstudio/eclipse/distribution/project/ui/wizard/DistributionProjectWizard.java

```diff
@@ -66,13 +67,19 @@ public IResource getCreatedResource() {
 
 	public boolean performFinish() {
 		try {
+			log.info("DistributionProjectModel creation");

```

Comment:
These are should not be "INFO" logs. They should be "DEBUG" logs. Please change the log level.